File: Sort Algorithms/selection.py
def selectionSort(array):
	for i in range(0, len(array)):
		minimum = i
		for j in range(i+1, len(array)):
			if array[j] < array[minimum]:
				minimum = j

		temp = array[minimum]
		array[minimum] = array[i]
		array[i] = temp

	return print(array)

selectionSort([6,5,0,1])

File: Sort Algorithms/selection.js
const selectionSort = (list) => {
	const convert = (list, i, j) => {
		[list[i], list[j]] = [list[j],list[i]]
	}

	for (let i = 0; i < list.length; i++) {
		let minimum = i
		for (let j = i+1; j < list.length; j++) {
			if (list[j] < list[minimum]) {
				minimum = j
			}
		}
		convert(list, i, minimum)
	}

	return console.log(list)
}

let sample = [6,5,4,3,2,1]
selectionSort(sample)